app/controllers/homes_controller.rb

```diff
@@ -39,58 +39,77 @@ def api_get(url)
     @lat = location.latitude
     @lng = location.longitude
 
-    @tweets = []
-    @local_trends = []
-    q = "geocode:39.5,-98.35,1500mi"
-    all_tweets = client.search(q).take(50)
-
-    if !all_tweets.nil?
-      @tweets = all_tweets.delete_if do |t|
-        /(#|)(job|hiring|work)/i.match(t.text)
-      end
-      raw = []
-      @tweets.each do |t|
-        words = t.text.split
-        words.each { |w| raw << w }
-      end
-      common_words = raw.select { |e| raw.count(e) > 2 && e.length > 3 }.uniq!
-      if !common_words.nil?
-        common_words.delete_if do |w|
-          /(have|this|with|just|your|when|&amp;|from|that|-&gt;|were)/i.match(w)
+    #################
+    sample_tweet = Tweet.first
+    if !sample_tweet.nil? || (sample_tweet != nil && ((sample_tweet.created_at + 3.minutes) < DateTime.now.utc))
```

Comment:
Line is too long. [112/80]<br>Prefer `!expression.nil?` over `expression != nil`.